modules/dev-tools/config/ocular.config.js

```diff
@@ -1,10 +1,18 @@
+const fs = require('fs');
```

Comment:
Not sure if this a default config? Never mind, my thought was that we now have two ocular modules that have ocular config files. I think the names are just slightly different, but assuming we want to keep the files separate (I think we should), we may want to distinguish them further. `ocular-dev-tools.config.js`? and `ocular-gatsby.config.js` @jckr